### DISCUSSION TITLE
USWDS tutorial

### DISCUSSION THREAD

**Discussion by matikin9** [uswds/uswds#4821] [General]
Hi! I just wanted to mention that there are 2 PRs on the tutorial repo to fix some minor typos.  :) 

Also, thanks for the step-by-step tutorial, it was really useful and easy to follow!

If I'm customizing the design system for my agency to use on multiple websites, do I understand correctly that I would do it as a new "theme"?

**Comment by mejiaj** [comment]
Hey @matikin9, thanks for submitting those PRs!

You _could_ use it as a new theme. Once you get more comfortable you can customize it further.
